src/components/Dashboard/TodoList.vue

```diff
@@ -453,8 +493,42 @@ export default {
     },
 
     loadCorrection (task) {
-      // eslint-disable-next-line no-console
-      console.log('loading correction not yet implemented')
+      const filing = task.task.filing
+      if (filing && filing.header && filing.correction) {
+        this.taskItems.push({
+          type: filing.header.name,
+          filingId: filing.header.filingId,
+          filingDate: filing.correction.correctedFilingDate,
+          corrFilingId: filing.correction.correctedFilingId,
+          correctedFilingType: this.formatFilingType(filing.correction.correctedFilingType),
+          title: `Priority Correction - ${this.formatFilingType(filing.correction.correctedFilingType)}`,
+          draftTitle: `Correction Filing`,
+          enabled: Boolean(task.enabled),
+          order: task.order,
+          comments: this.flattenAndSortComments(filing.header.comments)
+        })
+
+        // Flag to disable other filings
+        this.hasPendingCorrection = true
+      } else {
+        // eslint-disable-next-line no-console
+        console.log('ERROR - invalid filing or header or changeOfAddress in task =', task)
+      }
+    },
+
+    formatFilingType (filingType: string): string {
```

Comment:
This should look like typeToTitle() (which uses the FilingTypes enum for the strings). Or maybe, since this functionality is in 3 places, this should be in a mixin.

Reply:
Yea i agree, in a sense they are very similar. Might as well refactor this now!

Reply:
Oh, what is the 3rd spot? I know of something similar in the filing history and the todo list.

Reply:
The third place is the Correction filing page.

Reply:
Done and done, little bit of refactoring required 👍